testutil/testdata/testpb/unknonwnproto.proto

```diff
@@ -290,7 +290,7 @@ message TestUpdatedTxBody {
   repeated google.protobuf.Any messages                          = 1;
   string                       memo                              = 2;
   int64                        timeout_height                    = 3;
-  uint64                       some_new_field                    = 4;
+  uint64                       some_new_field                    = 5;
```

Comment:
@facundomedica why does this fix the issue?

Reply:
It's because a new field was added with tag 4 to `TxBody` which in `TestUpdatedTxBody` is `some_new_field`.

From what I understand, the rejecting of unknown fields happen by checking the tag numbers, so `TxBody` didn't have a tag 4 but because `TestUpdatedTxBody` did have it it was erroring (as expected). Now a field with tag 4 is expected, so to trigger the error of an unexpected field we have to modify the proto message used for this test.

Reply:
good find!! could you add a comment to not confuse future readers

Reply:
added a small comment on the test case 👌

Reply:
Yeah this was such a footgun -- never would've found out. Thanks!